salt/datadog/init.sls

```diff
@@ -1,3 +1,7 @@
+{% set swap_file = salt["pillar.get"]("swap_file", {}) %}
+{% set swap_size = swap_file.get("swap_size", "1024") %}
+{% set swap_path = swap_file.get("swap_path") %}
```

Comment:
```suggestion
{% set swap_path = salt["pillar.get"]("swap_file.swap_path") %}
```

not positive if this syntax is spot on, but this could simplify things to one line.

Reply:
I think its :